src/microbe_stage/Microbe.cs

```diff
@@ -789,21 +789,55 @@ public void Divide()
         copyEntity.Compounds.ClearCompounds();
 
         var keys = new List<Compound>(Compounds.Compounds.Keys);
+        var reproductionCompounds = CalculateTotalCompounds();
```

Comment:
This would be better to be called on `copyEntity` as that doesn't have the duplicated organelles in it, thought that doesn't break things, I still think that would be nicer
`var reproductionCompounds = copyEntity.CalculateTotalCompounds();`

Reply:
I changed this already but I don't think the player has the duplicates either since I believe it resets it before calling divide?

Reply:
I'm mainly commenting as we are interested specifically in what compounds the new cell needs for dividing, so it is clearer to ask that cell "hey, what do you need to divide?" than assume it is the same as for this.

Reply:
Yeah I get where you're coming from, though with how it's coded it technically clamps both cells just in case something crazy happened and the parent NPC cell ended up with a ton of compounds again.